CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+##[2.6.1] - 2020-10-22
+## Added
+- Cahngelog updates
```

Comment:
Small typo :D

Reply:
Delebartle mistakale, pwommise.

Thanks for spotting 😅  fixed now